goth/runner/probe.py

```diff
@@ -102,6 +115,22 @@ def name(self) -> str:
         """Name of the container."""
         return self.container.name
 
+    def _setup_gftp_proxy(self, config: YagnaContainerConfig) -> YagnaContainerConfig:
+        """Create a proxy script and a dir for exchanging files with the container."""
+
+        self._gftp_script_dir, gftp_volume_dir = create_gftp_dirs(config.name)
+        self._logger.info("Created gftp script at %s", self._gftp_script_dir)
+
+        new_config = copy.deepcopy(config)
+        new_config.volumes[gftp_volume_dir] = CONTAINER_MOUNT_POINT
```

Comment:
It would be clearer as to where this `CONTAINER_MOUNT_POINT` is coming from if we imported the entire `gftp` module instead of doing selective imports (i.e. `gftp.CONTAINER_MOUNT_POINT`).